src/Tribe/Attendees.php

```diff
@@ -729,4 +729,42 @@ public function user_can( $generic_cap, $event_id ) {
 
 		return false;
 	}
+
+	/**
+	 * Determines if the current user (or an ID-specified one) is allowed to delete, check-in, and
+	 * undo check-in attendees.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $user_id Optional. The ID of the user whose access we're checking.
+	 * @return boolean
+	 */
+	public function user_can_manage_attendees( $user_id = 0 ) {
+
+		$user_id = 0 === $user_id ? get_current_user_id() : $user_id;
+
+		if ( ! $user_id ) {
+			return false;
+		}
+
+		/**
+		 * Allows customizing the caps a user must have to be allowed to manage attendees.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $default_caps The caps a user must have to be allowed to manage attendees.
+		 */
+		$required_caps = apply_filters( 'tribe_tickets_caps_can_manage_attendees', array(
```

Comment:
Hey @ggwicz - I should have flagged this first time (though, it's not essential and could be added later) but maybe it would be sensible to pass the `$user_id` to callbacks, so they can more readily make case-by-case exceptions.

Or, for similar reasons, add a final `tribe_tickets_USER_can_manage_attendees` hook at the end of the method?

Again, not a block to merging.

Reply:
passing `user_id` as a param seems the best call, sorry for overlooking! adding now